test/functional/provider/node_spec.lua

```diff
@@ -15,3 +19,22 @@ end
 before_each(function()
   clear()
 end)
+
+describe('node host', function()
```

Comment:
Let's use "nodejs" everywhere possible. Reduces ambiguity when grepping, etc.

Reply:
"node" is used in https://github.com/neovim/neovim/blob/master/runtime/autoload/provider/node.vim and `rplugin/node`.

Reply:
Yes. I just mean all of the new instances in this PR. I'll change the other instances later, except for anything that might break clients.